File: Template/Python/CharParent.py
from Parent import Parent


class CharParent(Parent):
    def __init__(self, char):

        self.char = char

    def openP(self):
        print("Open Char Parent")

    def close(self):
        print("\nClose Char Parent")

    def println(self):
        print(self.char, end='')

File: Template/Java/src/CharParent.java
public class CharParent extends Parent{
    private char letter;
    public CharParent(char letter){
        this.letter = letter;
    }
    @Override
    public void open() {
        System.out.print("<<<");
    }

    @Override
    public void close() {
        System.out.print(">>>");
    }

    @Override
    public void print() {
        System.out.print(this.letter);
    }
}
